web/quickfeed_service.go

```diff
@@ -348,6 +348,15 @@ func (s *QuickFeedService) DeleteGroup(ctx context.Context, in *connect.Request[
 	return &connect.Response[qf.Void]{}, nil
 }
 
+func (s *QuickFeedService) GetSubmission(_ context.Context, in *connect.Request[qf.SubmissionReviewersRequest]) (*connect.Response[qf.Submission], error) {
```

Comment:
Please add a doc comment that explain what this returns. We should also say where it is used.

I'm not entusiastic about adding `GetLastSubmission` to the db interface. Right now I'm unable to find a better alternative that keeps the database stuff in the database package. But let's fix the database stuff in another PR.